hstrat/serialization/_unpack_differentiae.py

```diff
@@ -10,23 +11,25 @@ def unpack_differentiae(
     packed_differentiae: str,
     differentia_bit_width: int,
 ) -> typing.Iterable[int]:
-    """Unpack a sequence of differentiae from into a compact, concatenated
-    base 64 representation.
+    """Unpack a compact, concatenated base 64 representation into
+    a sequence of differentiae.
```

Comment:
into a sequence with each represented as a distinct integer.

Reply:
Changed in 5eab3550